### DISCUSSION TITLE
recipes/lunches/quick-couscous-chickpea-salad/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1093] [Comments]
# recipes/lunches/quick-couscous-chickpea-salad/

A collection of my recipes.

https://nicholaswilde.io/recipes/lunches/quick-couscous-chickpea-salad/

**Comment by nicholaswilde** [comment]
couscous 1/3 cup = 60 g

**Reply by nicholaswilde** [reply]
dry
